source/scpd/quorum/QuorumTracker.d

```diff
@@ -70,6 +70,3 @@ public:
     /// returns the known quorum map of the entire network
     ref const(QuorumMap) getQuorum () const;
 }
-
-// should be 48, see #753
-static assert(QuorumTracker.sizeof == 16);
```

Comment:
Why is this removed and not fixed ?

Reply:
While I think that does not really serve any purpose, I'll try to rewrite it better.